tools/ci/taskcluster-run.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+
+import argparse
+import gzip
+import logging
+import os
+import shutil
+import subprocess
+
+browser_specific_args = {
+    "firefox": ["--install-browser", "--reftest-internal"]
```

Comment:
`--reftest-internal` is the default, I think.

Reply:
I think you're right

    tools/wptrunner/wptrunner/wptcommandline.py:    if kwargs["reftest_internal"] is None:
    tools/wptrunner/wptrunner/wptcommandline.py-        # Default to the internal reftest implementation on Linux and OSX
    tools/wptrunner/wptrunner/wptcommandline.py:        kwargs["reftest_internal"] = sys.platform.startswith("linux") or sys.platform.startswith("darwin")

I wanted to verify with the person who added this flag, and [it turns out that was you](https://github.com/web-platform-tests/wpt/commit/1e51e1722a294830c85866462757d6924de7bac4#diff-57f0e26181ff24ebd57b4bf3976b6260):

> Explicitly set Firefox to use the fast reftest runner.
>
> This should happen by default on Linux, but it doesn't hurt to be explicit.

Have you had a change of heart? Or should we keep the flag for the sake of explicitness?

Reply:
Let's remove the flag and aim for a situation where we can use the same flags for all browsers.

Reply:
You got it.